src/AppInstallerCommonCore/AppInstallerLogging.cpp

```diff
@@ -42,18 +45,13 @@ namespace AppInstaller::Logging
         case Channel::YAML: return "YAML";
         case Channel::Core: return "CORE";
         case Channel::Test: return "TEST";
+        case Channel::Log:  return "LOG";
```

Comment:
To log any errors when initializing the loggers.

Reply:
I don't think that needs its own channel; it should be able to use Core.

Reply:
It helps in not logging to TraceProvider but only log to FileLogger for all Logger Initialization failures to help us debug those scenarios. This check will be in Write method of DiagnosticLogger object - this will be in the new commit - missed this earlier.